packages/zod/src/zod.test.ts

```diff
@@ -34,7 +34,11 @@ const queryParams: ZodValidationSchemaDefinitionInput = {
 describe('parseZodValidationSchemaDefinition', () => {
   describe('with `override.coerceTypes = false` (default)', () => {
     it('does not emit coerced zod property schemas', () => {
-      const parseResult = parseZodValidationSchemaDefinition(queryParams);
+      const parseResult = parseZodValidationSchemaDefinition(
```

Comment:
I am also updating the unit tests, but at the same time, I thought it would be a good idea to add a test that the options added this time work using the automatically generated source code of `zod` in the `tests` directory.

Reply:
I am not sure to understand. I added a test for the new option too. And you want to test the generated files also right? If so indeed would be great

Reply:
Yes, it is a good idea to continuously test that the automatically generated source code does not break when the options added this time are specified. And it's now in the `tests` directory.

Reply:
I added a test with the option

Reply:
Thanks!